src/main/java/com/adyen/Client.java

```diff
@@ -139,6 +144,25 @@ public void setEnvironment(Environment environment, String liveEndpointUrlPrefix
         }
     }
 
+    /**
+     * @param region The region for which the endpoint is requested. If null or the region is not found, defaults to default EU endpoint.
+     */
+    public String getCloudEndpoint(Region region, Environment environment) {
```

Comment:
The `getCloudEndpoint` method is not used when the `Client` is instantiated. See `setEnvironment(..)` function

Reply:
I'm not sure I understand you correctly. The `getCloudEndpoint` is an optional way to determine your regional endpoint. If we were to adjust `setEnvironment` we have to add the region parameter to the method.